### ISSUE TITLE
require('gulp-webdriver') fails

### ISSUE THREAD

**Issue by cristian-smocot** [webdriverio/gulp-webdriver#6]
I installed the plugin using `npm install gulp-webdriver --save-dev`, and then when I try to use it in my code, I get:

```
    module.js:338
    throw err;
          ^
Error: Cannot find module 'gulp-webdriver'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\Projects\test\selenium\testBrowser.js:4:17)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17) 
```

Looking in `node_modules\gulp-webdriver` folder, apparently `index.js` is not present.

Looking at `package.json`, the property `"main": "Gruntfile.js"` apparently is incorrect.
Shouldn't it be `index.js`?

**Comment by jimschubert** [comment]
I encountered the same thing. You can install repos directly using `owner/repo` syntax:

```
npm install webdriverio/gulp-webdriver --save-dev
```

Afterward, you'll be able to require the module.

**Comment by christian-bromann** [comment]
hi @cristian-smocot ,
yeah sorry, this project isn't ready yet. There are some tiny tweaks necessary to publish it. I am pretty close to that thanks to @jimschubert. I will let you know when I published the working version to NPM.
Cheers

**Comment by christian-bromann** [comment]
Just published `v0.1.1` to NPM. The plugin should now work as expected @cristian-smocot
